biobalm/_sd_algorithms/expand_source_SCCs.py

```diff
@@ -1,42 +1,30 @@
 from __future__ import annotations
 
 import itertools as it
-import sys
 from typing import TYPE_CHECKING, Callable, cast
 
-from biodivine_aeon import BooleanNetwork, SymbolicContext
-
-from biobalm._sd_algorithms.expand_bfs import expand_bfs
-from biobalm.space_utils import percolate_network, percolate_space
+from biobalm.space_utils import percolate_network
 from biobalm.types import BooleanSpace
+from biobalm.interaction_graph_utils import source_nodes
 
 if TYPE_CHECKING:
     from biobalm.succession_diagram import SuccessionDiagram
 
-    ExpanderFunctionType = Callable[
-        [SuccessionDiagram, int | None, int | None, int | None],
-        bool,
-    ]
-
-sys.path.append(".")
-
-DEBUG = False
+    ExpanderFunctionType = Callable[[SuccessionDiagram], bool]
 
 
 def expand_source_SCCs(
     sd: SuccessionDiagram,
-    expander: ExpanderFunctionType = expand_bfs,
-    check_maa: bool = True,
+    check_maa: bool,
```

Comment:
Ah, I forgot to put it back. I was playing with it when I started using the function recursively because it seemed like the flag wasn't propagating correctly. I'll put it back.